dev-util/shc/shc-3.9.6.recipe

```diff
@@ -0,0 +1,48 @@
+SUMMARY="A generic shell script compiler"
+DESCRIPTION="Shc takes a script, which is specified on the command line and \
+produces C source code. The generated source code is then compiled \
+and linked to produce a stripped binary executable. \
+The compiled binary will still be dependent on the shell specified in the \
```

Comment:
```diff
-produces C source code. The generated source code is then compiled \
+produces C source code. The generated source code is then compiled and linked \
-and linked to produce a stripped binary executable. \
+to produce a stripped binary executable. \
```

Reply:
I know we have the 80 char limit, but is it needed to fill out the lines to meet the limit?
PS, no problem to change it :)

Reply:
I hope I'm not being too picky, but if you are inside a sentence, then there is no reason to keep a short line.
That said, if you start a new sentence then, for me, it's OK to have short lines that end with a full stop or with a full stop and a backslash. (This allows e.g. to insert new sentences with less "diff-noise".)

We don't want to see recipes with unnecessarily short line and without even a tiny reason for that, don't we?

Reply:
Fair enough :+1: